### PR TITLE
three minor additions, including fix for logind abort

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: TODO [hunk 1]
```diff
@@ -32,6 +32,10 @@ Janitorial Clean-ups:
 
 Features:
 
+* add a percentage syntax for TimeoutStopSec=, e.g. TimeoutStopSec=150%, and
+  then use that for the setting used in user@.service. It should be understood
+  relative to the configured default value.
```
Comment (keszybz) [inline TODO:37]: by default you mean the DefaultTimeoutSec value, not the default for TimeoutSec?

Comment (poettering) [reply TODO:37]: not sure what you mean?

Comment (poettering) [reply TODO:37]: are you referring to the difference between DefaultTimeoutStopSec= value, and the default for the DefaultTimeoutStopSec= value? of course i mean, relative to the actual DefaultTimeoutStopSec= value in effect, which means 90s if nothing is configured there.

i.e. in the stop timeout hierarchy of defaults we have: TimeoutStopSec= → DefaultTimeoutStopSec= → 90s, and I think the relative setting in the first level should be taken towards the setting of the second level, if you follow what i try to say. It should not skip this second level, and be taken relative to the third level directly...

Comment (keszybz) [reply TODO:37]: Yeah.

**Pull request merged**